StackTraceExplorer.Tests/MemberRegexTests.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System.Linq;
 using System.Text.RegularExpressions;
+using FluentAssertions;
```

Comment:
FluentAssertions changed licenses, I want to stay away from this package to avoid any trouble. Please use Nunit or a fork.

Reply:
I was not aware of that. Thanks for the heads-up. Such a bad move.
I picked the community driven fork. Good enough at the moment. It will be easy to move on if something bad happen in the future...